sqlmesh/dbt/basemodel.py

```diff
@@ -109,8 +119,7 @@ class BaseModelConfig(GeneralConfig):
     storage_format: t.Optional[str] = None
     path: Path = Path()
     target_schema: str = ""
-    _dependencies: Dependencies = Dependencies()
-    _variables: t.Dict[str, bool] = {}
```

Comment:
@crericha this doesn't seem to be used anywhere so I dropped it. Let me know if this is wrong.

Reply:
Yeah, it got encapsulated into dependencies.